### ISSUE TITLE
Quantization code performance.

### ISSUE THREAD

**Issue by wonderzy** [tiny-dnn/tiny-dnn#552]
I need to use this on arm. I read the tiny_quantized_conv2d_kernel code, but it seems a brute_force one without using gemmlowp. Meanwhile, tiny-dnn seems only support the fc layer with gemmlowp. How can this play an amazing performance?

**Comment by bhack** [comment]
Can you join on https://github.com/tiny-dnn/tiny-dnn/pull/443?

**Comment by Randl** [comment]
Probably https://github.com/tiny-dnn/tiny-dnn/issues/551 is related too

**Comment by wonderzy** [comment]
What's the performance of gemmlowp vs Eigen. I have done some experiment with Tensorflow demo on Android but sadly found gemmlowp is twice worce than Eigen. Do you have such results, looking forward to response here.@bhack@Randl

**Comment by bhack** [comment]
@wonderzy Have you checked https://github.com/tensorflow/tensorflow/issues/2807?

**Comment by wonderzy** [comment]
@bhack Yes,  I got my result on Android not on PC. Some people also encounter this problem in the discussion.
But gemmlowp is optimized on ARM CPU, so the result I got is so confused.

**Comment by bhack** [comment]
I don't know if @wangyida benchmarked performance in tiny-dnn.

**Comment by wonderzy** [comment]
Hi, @bhack I know tiny-dnn use nnpack. I wonder if there is any experiment on speed of nnpack vs eigen on arm.

**Comment by bhack** [comment]
We don't have an eigen backend so you can try to ping https://github.com/Maratyszcza/NNPACK/issues/30
